tasks/task11.test.js

```diff
@@ -0,0 +1,12 @@
+const {say} = require('./task11');
+
+describe('Test #11', () => {
+    describe('say', () => {
+        test('should say("Hello, ")("it\'s me") to be "Hello, it\'s me"', () => {
```

Comment:
Consider making spec descriptions readable, not to include code inside -- like a business requirement. Simply:
```
test("should return a function that joins given string with original", () => {...})
```